docker/img2tar.sh

```diff
@@ -22,7 +22,8 @@ cd "$mount"
 #tar -cpSf - --acls --selinux --xattrs * | bzip2 > ${image}.tar.bz2
 
 # This one appears to work fine for docker creation
-tar --numeric-owner -c . | bzip2 > ${image}.tar.bz2
+# hacked by jefby,change the output file to /tmp/xxx.tar.bz2
+tar --numeric-owner -c . | bzip2 > /tmp/${image}.tar.bz2
```

Comment:
1. Why is this necessary?

2. This makes it impossible to use img2tar.sh with a qualified (fully or otherwise) path, e.g.:

    img2tar.sh foo/bar.img
    img2tar.sh /tmp/foo/img

3. If this really honestly needs to go in /tmp then you need to use a proper tmpfile (``mktemp``) to avoid exposing a known-tempfile attack.